### PR TITLE
Release v1.45.0

### PR DESCRIPTION
**Pull request by 0xaravindh**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, press enter to put that in a new line, and remove leading whitespace from that line:
>
> /kind breaking
> /kind bug
> /kind cleanup
> /kind documentation
> /kind feature
> /kind hotfix

 /kind release

**What this PR does / Why we need it**:

**Which issue(s) this PR fixes**:

Work On #4039

**Special notes for your reviewer**:

### CODE REVIEW

File: site/content/en/blog/releases/1.45.0.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+- **Terraform Scripts and Documentation for OKE Cluster Setup**: New documentation has been added, including Terraform scripts to help users set up an OKE (Oracle Kubernetes Engine) cluster and install Agones.
+
+- **Rust SDK Updates**: All Rust SDK dependencies have been updated to the latest versions. This update brings performance improvements, bug fixes, and ensures compatibility with the newest features in Rust.
+
+- **Fleet Autoscaler Enhancements: List-based Scaling from Zero Replicas**: The Fleet Autoscaler now supports scaling game server fleets from zero replicas when using a type List Fleet Autoscaler. This improves the flexibility of scaling fleets with GitOps based tools.
+
+- **New Dashboard Feature for GameServer State Duration**: A new dashboard feature has been added that provides insights into the duration of GameServer states. This will help operators monitor and debug the lifecycle of game servers more efficiently.
+
+Check the <a href="https://github.com/googleforgames/agones/tree/release-1.45.0" >README</a> for details on features, installation and usage.
```
Comment (igooch) [inline site/content/en/blog/releases/1.45.0.md:21]:
```suggestion
Check the <a href="https://github.com/googleforgames/agones/tree/release-1.45.0" data-proofer-ignore>README</a> for details on features, installation and usage.
```

File: site/content/en/blog/releases/1.45.0.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+**Breaking changes:**
+- Update Supported Kubernetes to 1.29, 1.30, 1.31 by @kamaljeeti in https://github.com/googleforgames/agones/pull/4024
+
+**Implemented enhancements:**
+- Dashboard for Agones GameServer State duration by @vicentefb in https://github.com/googleforgames/agones/pull/3947
+- Add Shutdown Delay Seconds to the sdk-client-test containers by @igooch in https://github.com/googleforgames/agones/pull/4030
+- Add a CI check to fail on change to an example without a new version by @wheatear-dev in https://github.com/googleforgames/agones/pull/3940
+
+See <a href="https://github.com/googleforgames/agones/blob/release-1.45.0/CHANGELOG.md" >CHANGELOG</a> for more details on changes.
```
Comment (igooch) [inline site/content/en/blog/releases/1.45.0.md:31]:
```suggestion
See <a href="https://github.com/googleforgames/agones/blob/release-1.45.0/CHANGELOG.md" data-proofer-ignore>CHANGELOG</a> for more details on changes.
```

File: site/content/en/blog/releases/1.45.0.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+- [us-docker.pkg.dev/agones-images/examples/nodejs-simple-server:0.10](https://us-docker.pkg.dev/agones-images/examples/nodejs-simple-server:0.10)
+- [us-docker.pkg.dev/agones-images/examples/rust-simple-server:0.13](https://us-docker.pkg.dev/agones-images/examples/rust-simple-server:0.13)
+- [us-docker.pkg.dev/agones-images/examples/simple-game-server:0.35](https://us-docker.pkg.dev/agones-images/examples/simple-game-server:0.35)
+- [us-docker.pkg.dev/agones-images/examples/supertuxkart-example:0.15](https://us-docker.pkg.dev/agones-images/examples/supertuxkart-example:0.15)
+- [us-docker.pkg.dev/agones-images/examples/unity-simple-server:0.3](https://us-docker.pkg.dev/agones-images/examples/unity-simple-server:0.3)
+- [us-docker.pkg.dev/agones-images/examples/xonotic-example:2.1](https://us-docker.pkg.dev/agones-images/examples/xonotic-example:2.1)
+
+Helm chart available with this release:
+
+- <a href="https://agones.dev/chart/stable/agones-1.45.0.tgz" >
```
Comment (igooch) [inline site/content/en/blog/releases/1.45.0.md:52]:
```suggestion
- <a href="https://agones.dev/chart/stable/agones-1.45.0.tgz" data-proofer-ignore>
```

**Review by igooch — CHANGES_REQUESTED** [review]
Did you run `make del-data-proofer-ignore FILENAME=1.45.0-1.md`? Let us know if that command is not working. It looks like `data-proofer-ignore` still needs to be removed from the agones/site/content/en/blog/releases/1.44.0.md file.

File: site/content/en/blog/releases/1.45.0.md (hunk 1 above)
Comment (0xaravindh) [reply site/content/en/blog/releases/1.45.0.md:21]: in release-1.45.0, there is no data-proofer-ignore right? 

this command is not working. "make del-data-proofer-ignore FILENAME=1.45.0-1.md" something we need to remove the word "-1" @igooch

**Review by igooch — APPROVED** [review]
LGTM

File: site/content/en/blog/releases/1.45.0.md (hunk 3 above)
Comment (0xaravindh) [reply site/content/en/blog/releases/1.45.0.md]: @igooch  thanks for you help!

**Pull request merged**